packages/core/src/agent/__tests__/Agent.test.ts

```diff
@@ -84,7 +84,7 @@ describe('Agent', () => {
       expect(wallet.isInitialized).toBe(false)
 
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      await wallet.initialize(walletConfig!, walletCredentials!)
+      await wallet.initialize(walletConfig!)
```

Comment:
Not quite sure what the flow exactly is here, but do we want a non-null-assertion or could we check it before?

Reply:
We know its defined, TypeScript doesn't. I don't think it's necessary to do for the test